book/Toys.ipynb

```diff
@@ -136,8 +136,8 @@
    "source": [
     "CLs_obs, CLs_exp = pyhf.infer.hypotest(\n",
     "    1.0,  # null hypothesis\n",
-    "    [5.0, 7.0] + model.config.auxdata,\n",
-    "    model,\n",
+    "    [5.0, 7.0] + model_low.config.auxdata,\n",
+    "    model_low,\n",
```

Comment:
Just above this is 

```python
model_low = pyhf.simplemodels.uncorrelated_background(
    signal=[0.5, 1.0], bkg=[5.0, 6.0], bkg_uncertainty=[0.5, 1.2]
)
```

but we were never using this `model_low` for some reason and so were trying to fit a model that terribly modeled the observations to begin with. This PR just used the model we meant to.